packages/taskrunner/src/main/java/software/uncharted/terarium/taskrunner/models/task/TaskResponse.java

```diff
@@ -16,6 +16,7 @@ public class TaskResponse implements Serializable {
 	private TaskStatus status;
 	private byte[] output;
 	private Object additionalProperties;
+	private String projectId;
```

Comment:
On both the models these should be typed `UUID` for projects to stay consistent - the typescript generated will stay as a `string` though!